README.md

```diff
@@ -1,122 +1,129 @@
+        <img alt="Mastodon Follow" src="https://img.shields.io/mastodon/follow/107278715447740005?color=3088d4&domain=https%3A%2F%2Fsocial.linux.pizza&label=Mastodon&logo=mastodon&logoColor=white&style=for-the-badge" loading="lazy" />
+    </a>
+    <a href="https://matrix.to/#/#pacstall:matrix.org"><img alt="join matrix" src="https://img.shields.io/matrix/pacstall:matrix.org?color=888888&label=Matrix&logo=Matrix&style=for-the-badge" /></a>
+    <br />
+    <!-- Link to the programs repository -->
+    <a href="https://github.com/pacstall/pacstall-programs">
+        <img alt="https://github.com/pacstall/pacstall-programs" src="https://img.shields.io/github/commit-activity/m/pacstall/pacstall-programs?style=for-the-badge&label=programs%20repo%20activity" />
+    </a>
 </p>
-<p align="center"><b>The AUR Ubuntu never had</b></p>
 
-<p align="center">Pacstall will attempt to become the AUR Ubuntu wishes it had. It takes the AUR and puts a spin on it, making it easier to install programs without scouring github repos and the likes.</p>
-<p align="center">The list of available packages can be found <a href="https://github.com/pacstall/pacstall-programs/tree/master/packages">here</a>.</p>
-</p>
+## What is this
 
----
+Pacstall is an AUR inspired package manager for Ubuntu. It takes the AUR and
+puts a spin on it, making it easier to install programs without scouring github
+repos and the likes.
 
-### Features
+You can find the list of avaiable packages to install
+[here](https://github.com/pacstall/pacstall-programs/tree/master/packages).
 
-*  Supports binary, git, appimage, building and `.deb` packages
-*  Accelerated package download using [axel](https://github.com/axel-download-accelerator/axel) (optional)
-*  Auto update checks for git packages, so you always get the latest build of your favourite program off the latest commit by the developer
-*  Over 175 packages available to install in the official programs repository
-*  Ability to install programs from multiple repositories
-*  Ability to track Pacstall updates from any fork/branch easily
-*  Completions available for `bash` (`ZSH`), and `fish`
+## Features
 
----
+* Supports [AppImage](https://appimage.org), binary, building, `.deb` and git
+  packages.
+* Asynchronous downloads.
+* Auto update checks for git packages, so you always get the latest build of
+  your favourite program off the latest commit by the developer.
+* Track multiple pacscript sources, instead of just the official one.
+* Completions available for `bash`,`fish`,`powershell`,`pwsh` and `zsh`.
 
-### Installing
+## Installation
 
-You can run the command below to install Pacstall.
-You can also grab the deb file [here](https://github.com/pacstall/pacstall/releases/latest) but it may be a bit older.
-```bash
-sudo bash -c "$(curl -fsSL https://git.io/JsADh || wget -q https://git.io/JsADh -O -)"
+To install the latest release run:
+
+```console
+$ pip install pacstall
```

Comment:
Shouldn't it be a system package?

Reply:
What do you mean? You want pacstall to be installed with `sudo`?

Reply:
Otherwise `sudo pacstall` wouldn't work, I think

Reply:
It can't find local commands
![image](https://user-images.githubusercontent.com/65723952/160139396-b42a96b1-175d-4c57-9a76-7fc83cac7858.png)

Reply:
So what do you want it to be? Suggest the changes.

Reply:
We would probably have to `sudo pip3 install` or install as a deb package